card/src/main/java/com/adyen/checkout/card/internal/ui/model/InputFieldUIState.kt

```diff
@@ -8,6 +8,6 @@
 
 package com.adyen.checkout.card.internal.ui.model
 
-internal enum class InputFieldUIState {
+enum class InputFieldUIState {
```

Comment:
I will only comment here but it applies everywhere, I think you initially made all the internal classes non internal but I think now that you can make them internal again, I suggest you go through them all and make sure only the ones that need to be public are made public and annotated with `@RestrictTo`

Reply:
Done, all of them needed to be `@RestictTo`